client/src/components/ApplicationForm/Dropzone.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useState, Dispatch, SetStateAction } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+type DropzoneProps = {
+  className: string;
+  setFile: Dispatch<SetStateAction<File | undefined>>;
```

Comment:
It's usually best to not use `setXxx` on callback props because that makes them look too much like state setters. The convention is to name them `onXxx` instead, which makes it more clear it's a prop.

In your case, `onFile` would be weird, but `onFileLoad` would be quite clear.